src/train_tracker.py

```diff
@@ -0,0 +1,228 @@
+"""列車捕捉モジュール.
+
+列車を捕捉する
+@author: desty505 miyashita64
+"""
+
+import cv2
+import numpy as np
+from camera_interface import CameraInterface
+
+
+class TrainTracker:
+    """列車を捕捉するクラス."""
+
+    def __init__(self, camera_id: int) -> None:
+        """コンストラクタ.
+
+        Args:
+            camera_id (int): カメラID
+        """
+        self.camera = CameraInterface(camera_id)
+        self.diff_border = 20
+        self.observe_rect_points = [(0, 0), (0, 0)]
+
+    def __del__(self) -> None:
+        """デストラクタ."""
+        del self.camera
```

Comment:
```suggestion
    def __del__(self) -> None:
        """デストラクタ."""
```
`del self.camera`はいらない